packages/app-admin/src/plugins/Menu/gqlApiInformation.tsx

```diff
@@ -26,17 +34,19 @@ const plugin: ApiInformationDialogPlugin = {
         return (
             <>
                 <div className={style.apiUrl}>
-                    <Tooltip
-                        className={style.api}
-                        content={
-                            <span>
-                                This link allows you to access content created by different
-                                application across Webiny like Page Builder or Form Builder.
-                            </span>
-                        }
-                    >
-                        <Typography use={"headline6"}>GraphQL API:</Typography>
-                    </Tooltip>
+                    <div className={style.api}>
+                        <Typography use={"headline6"} >GraphQL API:</Typography>
```

Comment:
Please remove `:`, so it's just:

> GraphQL API ℹ️

Looks a bit weird if we put the ℹ️  before `:`.